packages/renovate-config-cozy-konnector/package.json

```diff
@@ -19,8 +19,8 @@
           ],
           "timezone": "Europe/Paris",
           "schedule": [
-            "after 8pm every weekday",
-            "before 7am every weekday",
+            "every weekday after 08:00 pm",
+            "every weekday before 07:00 am",
```

Comment:
I disagree with your explanation of the solution.

In my understanding, the renovate PRs were opening on time.
But the automerge job was running out of the schedule.

We can try this solution, but I don't think that works